packages/lib-classifier/src/plugins/drawingTools/experimental/components/TranscriptionLine/TranscriptionLine.js

```diff
@@ -2,10 +2,18 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import { DragHandle } from '@plugins/drawingTools/components'
 
-const FINISHER_RADIUS = 8
+const FINISHER_RADIUS = 6
 const GRAB_STROKE_WIDTH = 6
+const COLOURS = {
+  active: '#5cb85c',
+  default: '#00ced1',
+  transcribed: '#c33',
+  complete: '#979797'
+}
 
-function TranscriptionLine ({ active, mark, onFinish, scale }) {
+function TranscriptionLine ({ active, mark, onFinish, scale, state }) {
+  state = active ? 'active' : state
+  const colour = COLOURS[state]
```

Comment:
Note that if state is somehow an invalid value, the colour of the T-Line will default to the tool's user-designated colour.